apps/stellar-contracts/tests/cli/utils/contract-addresses.sh

```
@@ -0,0 +1,78 @@
+    
+    init_contract_addresses
+    
+    # Update the JSON file
+    jq --arg name "$contract_name" --arg addr "$address" \
+       '.($name) = $addr' "$CONTRACT_ADDRESSES_FILE" > "$CONTRACT_ADDRESSES_FILE.tmp" && \
+    mv "$CONTRACT_ADDRESSES_FILE.tmp" "$CONTRACT_ADDRESSES_FILE"
+}
+
+# Update deployment timestamp
+update_deployment_timestamp() {
+    local timestamp=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)